pkg/api/client.go

```diff
@@ -85,6 +83,12 @@ type RESTClient interface {
 	// Put issues a PUT request to the specified path with the specified body.
 	// The response is populated into the response argument.
 	Put(path string, body io.Reader, response interface{}) error
+
+	// Raw issues a request with type specified by method to the
+	// specified path with the specified body.
+	// The response is returned rather than being populated
+	// into a response argument.
+	Raw(method string, path string, body io.Reader) (*http.Response, error)
```

Comment:
I am not in love with this name, I am open to suggestions on a better one.

Reply:
Ideally it would be something like `Do` to indicate the low-levelness of this command, but that's already taken 😅 I'm open to something generic like "Request". Agreed that "raw" is unclear here